bootstrap/kubeadm/internal/cloudinit/cloudinit.go

```diff
@@ -27,6 +27,7 @@ import (
 
 const (
 	standardJoinCommand            = "kubeadm join --config /run/kubeadm/kubeadm-join-config.yaml %s"
+	sentinelFileCommand            = "echo success > /run/cluster-api/bootstrap-success.complete"
```

Comment:
```suggestion
	sentinelFileCommand            = "touch /run/cluster-api/bootstrap-success.complete"
```

Maybe?

Reply:
unfortunately `touch` won't run on windows

Reply:
😬 , sounds good then. Do we need to write a string to it? Would an empty string suffice here?

Reply:
I think it's probably best, on Windows `echo > file.txt` would result in a file with the contents `ECHO is on.` :neutral_face:

Reply:
^ what James said

Reply:
I'll add a comment so it's clear

Reply:
😂 Sounds good, TIL